src/components/Comments/CommentBlockComponent.jsx

```diff
@@ -0,0 +1,254 @@
+import React, { Component } from 'react';
+import { Icon } from 'react-materialize';
+import PropTypes from 'proptypes';
+import swal from 'sweetalert';
+import authUser from '../../utils/authUser.util';
+import avatar from '../../assets/img/avatar.png';
+import '../../assets/styles/Comments.scss';
+
+class Comment extends Component {
+  constructor(props) {
+    super(props);
+    const { body } = this.props;
+    this.defaultState = {
```

Comment:
Why not just define this in the state object?

Reply:
but there are conditions I want them set to default state despite the current state of the component

Reply:
Since you are just spreading the default state in state, then I dont see the point of having the constant